package.json

```diff
@@ -20,7 +24,7 @@
     "object",
     "array"
   ],
-  "main": "objectAssignDeep.js",
+  "main": "bin/objectAssignDeep.js",
```

Comment:
If possible, I'd like to keep the main export as the original ES6 source file, as the transpiled ES5 code will be significantly slower.

I haven't tested this but it might be possible to include a `main.js` file with the following code:

```javascript
module.exports = require(
  process && process.title === 'node' && parseInt(process.versions.node.split('.')[0]) >= 6 ? './src/objectAssignDeep.js' : './bin/objectAssignDeep.js'
);
```